asreview/webapp/src/PreReviewComponents/ListItemPaper.js

```diff
@@ -85,7 +85,7 @@ const ListItemPaper = (props) => {
         fullWidth={true}
       >
         <DialogTitleWithClose
-          title={"Is this article relevant?"}
+          title={"If this document can be used as prior knowledge, is it relevant or irrelevant?"}
```

Comment:
@J535D165 What is the maximum of characters I can use?

Reply:
![image](https://user-images.githubusercontent.com/17449217/104128148-2b1ca800-5366-11eb-8dab-400e081b76db.png)

It looks too long as a title...

Reply:
@MartijnUX Do you have suggestions?